packages/yoga/src/Card/native/EventCard/__snapshots__/EventCard.test.jsx.snap

```diff
@@ -1026,6 +1026,51 @@ exports[`<EventCard /> Snapshots should match snapshot of small EventCard active
                     },
                   },
                 },
+                "snackbar": Object {
```

Comment:
Do we need to update the `EventCard` snapshots?

Reply:
Unfortunately, @ericcleao. 

It is an issue that should be addressed in other PR

Reply:
It was avoiding me to push changes to remote. I asked @ggdaltoso what should I do, and he suggested updating those snapshots